### PR TITLE
noob trying to get his PR accepted

### PR DESCRIPTION
**Pull request by Malhar-06**

### CODE REVIEW

File: _haikus/Fragrance.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Hacktoberfest2022
+author: Malhar Chikhale
+---
+
+    Her name is Fragrance
```
Comment (ltagliaferri) [inline _haikus/Fragrance.md:7]:
```suggestion
Her name is Fragrance
```

File: _haikus/Fragrance.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    Spreading Freshness is her job
```
Comment (ltagliaferri) [inline _haikus/Fragrance.md:8]:
```suggestion
Spreading Freshness is her job
```

File: _haikus/Fragrance.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    That is her Power
```
Comment (ltagliaferri) [inline _haikus/Fragrance.md:9]:
```suggestion
That is her Power
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Can you remove the trailing space from the front of each line please? Thank you.

File: _haikus/Fragrance.md (hunk 3 above)
Comment (Malhar-06) [reply _haikus/Fragrance.md:9]: ofcourse

**Review by ltagliaferri — APPROVED** [review]

**Comment by ltagliaferri** [comment]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**